pixi.toml

```diff
@@ -38,6 +38,7 @@ examples = { cmd = [
     "--verbose",
     "--junitxml=examples_report.xml",
 ], depends_on = ["install"], cwd = "imod/tests" }
+pypi-publish = { cmd = "rm --recursive --force dist && python -m build && twine check dist/*" }
```

Comment:
Maybe add `hatchling` to the dependencies?

Reply:
Good catch. I've added it to the pyproject dev dependencies.

Reply:
By the way, we could arguably remove the dev dependencies since we recommend pixi anyway, right?

And hatchling then wouldn't be need for a regular user.

Reply:
Yeah, that should be fine.

Reply:
Still annoying that there is no good way to find out if you require too much or especially too little in your normal dependencies.